### PR TITLE
DCS-1932: ✨Add Base Page For `New Arrival Summary Page` Part 1

### PR DESCRIPTION
**Pull request by connormaglynn**
Just adding the base components for the new /summary page.

Still need to add in:
- breadcrumbs
- additional data
- integration testing
- wiring up the route into the flow

but this seemed like a neat place to break off 😄

### CODE REVIEW

File: server/views/pages/bookedtoday/summary.njk [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+            <div class='govuk-grid-row'>
+                <div class='govuk-grid-column-two-thirds'>
+                    {{ summaryCard({
+                        classes: "govuk-!-margin-bottom-6",
+                        titleText: "Booking overview",
+                        rows: [
+                            {
+                                key: {  text: "Arrival Status" },
+                                value: { html: "<strong class=\"govuk-tag govuk-tag--grey\">" + arrival.potentialMatches[0].arrivalType + "</strong>" }
```
Comment (andrewrlee) [inline server/views/pages/bookedtoday/summary.njk:29]: This will always be "NOT ARRIVED" for pre arrival.

Comment (connormaglynn) [reply server/views/pages/bookedtoday/summary.njk:29]: Awesome, saves me having to map the types to strings! 😂

File: server/views/pages/bookedtoday/summary.njk [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+{% set pageTitle = "Person booked to arrive" %}
+
+{% block content %}
+    <div class="govuk-!-margin-bottom-8">
+
+        <div class="govuk-body">
+            {% if arrival.prisonNumber or arrival.pncNumber %}
+                <span class="govuk-caption-l">
+                    {% if arrival.prisonNumber %}Prison number: {{ arrival.prisonNumber }}{% endif %}{% if arrival.prisonNumber and arrival.pncNumber %} | {% endif %}{% if arrival.pncNumber %}PNC: {{ arrival.pncNumber }}{% endif %}
+                </span>
```
Comment (andrewrlee) [inline server/views/pages/bookedtoday/summary.njk:13]: could possibly use [join](https://mozilla.github.io/nunjucks/templating.html#join) here?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/routes/bookedtoday/summary.test.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { MatchType } from '../../services/matchTypeDecorator'
+
+let app: Express
+const expectedArrivalsService = createMockExpectedArrivalsService()
+
+const arrival = {
+  ...createArrival({
+    potentialMatches: [],
+  }),
+  matchType: MatchType.NO_MATCH,
```
Comment (andrewrlee) [inline server/routes/bookedtoday/summary.test.ts:17]: you can wrap this `withMatchType`

```js
const arrival = withMatchType(createArrival({
    potentialMatches: [],
  }))
```

```

Comment (andrewrlee) [reply server/routes/bookedtoday/summary.test.ts:17]: Though you wont even need potential matches 
```js
const arrival = withMatchType(createArrival())
```

**Pull request merged**